pkg/container-collection/containers.go

```diff
@@ -65,6 +65,12 @@ type Container struct {
 	PodUID    string            `json:"podUID,omitempty"`
 
 	ownerReference *metav1.OwnerReference
+
+	// We keep an open file descriptor of the containers mount namespace to be sure the kernel
+	// doesn't reuse the inode id before we get rid of this container. This logic avoids a race
+	// condition when the mnt ns inode id is reused by a new container and we erroneously pick
+	// events from it.
+	mntNsFd int
```

Comment:
It is not obvious that this field is only used with `WithTracerCollection` when `cc.cachedContainers != nil`. Could you add it in the comment or alternatively initialize the field to `-1`?

Reply:
Sure. Done!